src/jubjub/mod.rs

```diff
@@ -39,6 +40,7 @@ pub trait JubjubEngine: Engine {
 pub trait JubjubParams<E: JubjubEngine>: Sized {
     fn edwards_d(&self) -> &E::Fr;
     fn montgomery_a(&self) -> &E::Fr;
+    fn montgomery_2a(&self) -> &E::Fr;
```

Comment:
I think we should use ``A``, not ``a``. ``a`` is also a twisted Edwards parameter (equal to -1 for Jubjub), which could be confusing.

Reply:
Also add a comment that this implementation only supports twisted-Edwards a = -1, and Montgomery B = 1.

Reply:
I agree, I think I encountered some name conventions/warnings with the rust compiler when it comes to uppercase names like that.

Reply:
This is why I explicitly say `montgomery_` beforehand, versus using `A` which the rust compiler complains about (not snake-case).